src/features/houseHold/actions/houseHolds.ts

```diff
@@ -1,27 +1,30 @@
 "use server";
 import { z } from "zod";
-import { houseHoldSchema } from "../schema/houseHolds";
+import { householdSchema } from "../schema/households";
 import { redirect } from "next/navigation";
-import { insertHouseHold as insertHouseHoldDB } from "../db/houseHolds";
+import { insertHousehold as insertHouseholdDB } from "../db/households";
 import { auth } from "@/lib/auth";
 
-export async function insertHouseHold(
-  unsafeData: z.infer<typeof houseHoldSchema>,
+export async function insertHousehold(
+  unsafeData: z.infer<typeof householdSchema>,
   locale: string
 ) {
-  const { success, data } = houseHoldSchema.safeParse(unsafeData);
+  const { success, data } = householdSchema.safeParse(unsafeData);
 
   if (!success) throw new Error("Failed to create Household");
 
   const session = await auth();
 
   if (session?.user.id == null) throw new Error("User not found");
 
-  const houseHold = await insertHouseHoldDB({
-    ...data,
-    description: data.description != "" ? data.description : null,
-    ownerId: session?.user.id,
-  });
+  const houseHold = await insertHouseholdDB(
+    {
+      ...data,
+      description: data.description != "" ? data.description : null,
```

Comment:
to możesz walidować na wejściu sprawdzając zodem tak aby description było albo nullem albo niepustym stringiem